automated_qc/tree_cover_loss_qc.py

```diff
@@ -0,0 +1,132 @@
+import os
+import math
+import numpy as np
+import rasterio as rio
+import fiona
+import pandas as pd
+import argparse
+from utils import parse_bounds, concatenate_windows
+from tqdm import tqdm
+from rasterio.windows import from_bounds
+from rasterstats import zonal_stats
+
+# parse arguments
+parser = argparse.ArgumentParser(description='Calculate tree cover loss for admin area')
+parser.add_argument('--admin_area', type=str, help='path to admin area shapefile')
+parser.add_argument('--treecover_threshold', type=int, help='percent threshold for tree cover loss')
+args = parser.parse_args()
+shp_fp = args.admin_area
+treecover_threshold = args.treecover_threshold
+
+# get admin area bounds
+with fiona.open(shp_fp) as src:
+    bounds = src.bounds
+X_list, Y_list = parse_bounds(bounds)

```

Comment:
very minor, but in Python the style is typically that all variables should be lower case!